Conceptpower+Spring/pom.xml

```diff
@@ -9,7 +9,7 @@
 	<version>2.0-BUILD-SNAPSHOT</version>
 	<properties>
 		<java-version>1.8</java-version>
-		<org.springframework-version>4.3.18.RELEASE</org.springframework-version>
+		<org.springframework-version>4.3.7.RELEASE</org.springframework-version>
```

Comment:
is there a reason you downgraded this?

Reply:
In one of the previous tasks, I believe I saw 4.3.7. That's why I switched to that version. Could be my bad. I'll restore back.